src/components/about-chat-sidebar/AboutChatSidebar.tsx

```diff
@@ -0,0 +1,144 @@
+  files: File[]
+  links: Link[]
+}
+
+const AboutChatSidebar: FC<AboutChatSidebarProps> = ({
+  user,
+  media,
+  files,
+  links
+}) => {
+  const { t } = useTranslation()
+  const navigate = useNavigate()
+  const [isOpened, setIsOpened] = useState(true)
+
+  const {
+    _id,
+    firstName,
+    lastName,
+    photo,
+    role,
+    professionalSummary: userDescription
+  } = user
+
+  const userProfileURL = createUrlPath(authRoutes.userProfile.path, _id, {
+    role
+  })
+
+  const closeSidebar = () => {
+    setIsOpened(false)
+  }
+
+  return (
+    <Box
+      data-testid='sidebar'
+      style={isOpened ? undefined : { display: 'none' }}
+      sx={styles.wrapper}
+    >
+      <Box sx={styles.header}>
+        <Typography sx={spliceSx(styles.header.text, styles.title)}>
+          {t(`chat.sidebar.about`)}
+        </Typography>
+        <IconButton
+          aria-label='close'
+          onClick={closeSidebar}
+          sx={styles.header.iconButton}
+        >
+          <Close />
+        </IconButton>
+      </Box>
+      <Divider />
+      <Box sx={styles.chatInfo}>
+        <Avatar src={photo} sx={styles.chatInfo.avatar} />
+        <Typography sx={styles.title}>{`${firstName} ${lastName}`}</Typography>
+        <AppButton
+          onClick={() => navigate(userProfileURL)}
+          size={SizeEnum.Medium}
+          sx={styles.secondaryText}
+          variant={ButtonVariantEnum.Tonal}
+        >
+          View Profile
+        </AppButton>
+        <Typography sx={styles.chatInfo.description}>
+          {userDescription || t(`chat.sidebar.noSummary`)}
+        </Typography>
+      </Box>
+      <Divider />
+      <SidebarContentBox Icon={ImageOutlinedIcon} name={'Media'}>
+        {media.length !== 0 ? (
+          <SidebarImageGrid images={media} />
+        ) : (
+          <Typography sx={styles.notFound}>
+            {t(`chat.sidebar.noMedia`)}
+          </Typography>
+        )}
+      </SidebarContentBox>
+      <Divider />
+      <SidebarContentBox
+        Icon={InsertDriveFileOutlinedIcon}
+        content={files}
+        name={'Files'}
+      >
+        <Box sx={styles.verticalGrid}>
+          {files.length !== 0 ? (
+            files
+              .slice()
+              .reverse()
+              .map((file) => <FileComponent file={file} key={file._id} />)
+          ) : (
+            <Typography sx={styles.notFound}>
+              {t(`chat.sidebar.noFiles`)}
+            </Typography>
+          )}
+        </Box>
+      </SidebarContentBox>
+      <Divider />
+      <SidebarContentBox Icon={LinkOutlinedIcon} content={links} name={'Links'}>
+        <Box sx={styles.verticalGrid}>
+          {links.length !== 0 ? (
+            links
+              .slice()
+              .reverse()
+              .map((link) => <LinkComponent key={link._id} link={link} />)
+          ) : (
+            <Typography sx={styles.notFound}>
+              {t(`chat.sidebar.noLinks`)}
+            </Typography>
```

Comment:
And here move to a variable. Also, delete slice and reverse

Reply:
> And here move to a variable. Also, delete slice and reverse

fixed